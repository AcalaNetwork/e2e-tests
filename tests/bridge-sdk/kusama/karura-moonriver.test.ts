import { bridgeSDKTest } from '../shared'

bridgeSDKTest({
  from: 'karura',
  to: 'moonriver',
  token: 'KAR',
})
