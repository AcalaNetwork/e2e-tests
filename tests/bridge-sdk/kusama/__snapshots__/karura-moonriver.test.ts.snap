// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`'karura' to 'moonriver' using bridgeSDK cross-chain 'KAR' > after 1`] = `
{
  "address": "0x4E7440dB498561A46AAa82b9Bc7d2D5162b5c27B",
  "decimals": 18,
  "fromChain": "(rounded 8)",
  "toChain": "(rounded 2.06)",
}
`;

exports[`'karura' to 'moonriver' using bridgeSDK cross-chain 'KAR' > fee 1`] = `"(rounded 0.03)"`;

exports[`'karura' to 'moonriver' using bridgeSDK cross-chain 'KAR' > initial 1`] = `
{
  "address": "0x4E7440dB498561A46AAa82b9Bc7d2D5162b5c27B",
  "decimals": 18,
  "fromChain": 10,
  "toChain": 0.09012,
}
`;
